root/etc/fail2ban/filter.d/urbackup-auth.conf

```diff
@@ -2,5 +2,5 @@
 #this filter is made against brute force attack to the server of urbackup
 # Author Stephane de Labrusse <stephdl@de-labrusse.fr>
 
-failregex = ^<HOST> - - \[*\] "POST /{${'urbackup-server'}{'Name'}||'urbackup'}/x\?a\=login HTTP/1.1" 200 33$
+failregex = ^<HOST> - - \[*\] "POST /urbackup/x\?a\=login HTTP/1.1" 200 33$
```

Comment:
sysadmin can change the /urbackup, if you do that the jail won't catch it

Reply:
The file is now marked as configuration, the user can manually change it.

Reply:
yes but it is manual, sysadmin won't go to the terminal

Reply:
The fail2ban must not have access to urbbackup internals (like the prop `name`).
If this is still needed, the relevant code should be moved to a template inside the urbbackup package itself.